spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/mustache/web/MustacheViewResolver.java

```diff
@@ -39,8 +40,9 @@
 public class MustacheViewResolver extends UrlBasedViewResolver {
 
 	private Compiler compiler = Mustache.compiler();
+        private String charset = "UTF-8"; // default charset ?
 
```

Comment:
There is already a property in `MustacheProperties` (in the parent class, actually).